keps/sig-node/1029-ephemeral-storage-quotas/README.md

```diff
@@ -576,6 +579,71 @@ required elsewhere:
   future allow adding additional data without having to change code
   other than that which uses the new information.
 
+#### Implementation details of using User Namespace to Toggle Ephemeral-Storage-Quotas
+
+* The implementation centers around a feature toggle which 
+controls the integration of xfs-quota with user namespaces for 
+managing filesystem quotas within containers. The toggle allows 
+to switch between a quota management system and a traditional 
+disk usage monitoring method.
+
+* During container creation, there must be a check to verify 
+if xfs-quota in Kubelet is enabled. Additionally, verify if 
+hostUsers in pod is set to false, indicating that user namespaces 
+are enabled.
+
+* Only if both, xfs-quota and user namespace, are enabled then 
+the toggle enables xfs-quota on the container. If either xfs-quota 
+is not enabled or hostUsers is set to true, then the kubelet should 
+fallback to the legacy approach of using du and find commands to 
+periodically gather information about storage and inode consumption 
+of volumes
+
+### Risks and Mitigations
+
+* The SIG raised the possibility of a container being unable to exit
+  should we enforce quotas, and the quota interferes with writing the
+  log.  This can be mitigated by either not applying a quota to the
```

Comment:
Logs are written by a container monitor process to a directory which should't be under a quota so we need to clarify here.

Reply:
FYI this piece was just copy pasted from below. we can iterate here but just noting this isn't new

Reply:
Imposition of quota on log directory is the existing behavior
https://discuss.kubernetes.io/t/why-stdout-logs-are-accounted-in-ephemeral-storage-usage/27815